### PR TITLE
New delete-PVCs CronJob to delete old PVCs in sandbox

### PR DESCRIPTION
**Pull request by jpopelka**
Work-around for https://github.com/packit-service/packit-service/issues/409

### CODE REVIEW

**Comment by jpopelka** [comment]
Tested @ `stg`, ready for review

File: cron-jobs/delete-pvcs/delete-pvcs.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+set -x
+
+oc get pvc -n "packit-${DEPLOYMENT}-sandbox" | grep pvc |
+while IFS= read -r line; do
+  name=$(echo $line | awk '{print $1}')
+  age=$(echo $line | awk '{print $7}')
+  if [ "${age: -1}" == "d" ]; # days
+  then
+    echo "deleting ${name}"
+    oc delete pvc "${name}"
```
Comment (csomh) [inline cron-jobs/delete-pvcs/delete-pvcs.sh:14]: Please make the namespace explicit here too.

File: cron-jobs/delete-pvcs/delete-pvcs.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/bash
+
+oc login "${HOST}" --token="${TOKEN}" --insecure-skip-tls-verify
```
Comment (csomh) [inline cron-jobs/delete-pvcs/delete-pvcs.sh:3]: Why is `--insecure-skip-tls-verify` needed here?

**Comment by csomh** [comment]
> what I remember when this is happening, we tend to run out of quota within hours, not days

Pretty much all the times I was cleaning up storage I was deleting PVCs which were days old. So I would say the current solution is good enough, but I'm not against having a more aggressive approach.

File: cron-jobs/delete-pvcs/delete-pvcs.sh (hunk 1 above)
Comment (jpopelka) [reply cron-jobs/delete-pvcs/delete-pvcs.sh]: added

File: cron-jobs/delete-pvcs/delete-pvcs.sh (hunk 2 above)
Comment (jpopelka) [reply cron-jobs/delete-pvcs/delete-pvcs.sh]: removed

**Comment by TomasTomecek** [comment]
> > what I remember when this is happening, we tend to run out of quota within hours, not days
> 
> Pretty much all the times I was cleaning up storage I was deleting PVCs which were days old. So I would say the current solution is good enough, but I'm not against having a more aggressive approach.

oki, maybe things changed over time and the situation is better now (with days, not hours)

**Pull request merged**